<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPicker</class>
 <widget class="QWidget" name="ColorPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="ColorPickerHelpers::ColorPickArea" name="colorPickArea" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ColorPickerHelpers::ColorValueBar" name="valuePickBar" native="true">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ColorPickerHelpers::ColorShowWidget" name="colorShow" native="true">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ColorPickerHelpers::AlphaChannelBar" name="alphaChannelBar" native="true">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="colorEditLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="hsvLayout">
           <item>
            <layout class="QHBoxLayout" name="valLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="valLabel">
               <property name="text">
                <string>Val:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="valSpinBox">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="satLayout">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="satLabel">
               <property name="text">
                <string>Sat:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="satSpinBox">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hueLayout">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="hueLabel">
               <property name="text">
                <string>Hue:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="hueSpinBox">
               <property name="maximum">
                <number>359</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="rgbLayout">
           <item>
            <layout class="QHBoxLayout" name="redLayout">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="redLabel">
               <property name="text">
                <string>Red:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="redSpinBox">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="greenLayout">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="greenLabel">
               <property name="text">
                <string>Green:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="greenSpinBox">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="blueLayout">
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="blueLabel">
               <property name="text">
                <string>Blue:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="blueSpinBox">
               <property name="maximum">
                <number>255</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hexLayout">
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="hexLabel">
           <property name="text">
            <string>Hex:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="hexLineEdit">
           <property name="inputMask">
            <string>\#HHHHHH</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pickColorFromScreenButton">
           <property name="text">
            <string>Pick color from screen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_8">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerHelpers::ColorPickArea</class>
   <extends>QWidget</extends>
   <header>widgets/ColorPicker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPickerHelpers::ColorValueBar</class>
   <extends>QWidget</extends>
   <header>widgets/ColorPicker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPickerHelpers::ColorShowWidget</class>
   <extends>QWidget</extends>
   <header>widgets/ColorPicker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPickerHelpers::AlphaChannelBar</class>
   <extends>QWidget</extends>
   <header>widgets/ColorPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
